formatbibliography.sh

```diff
@@ -0,0 +1 @@
+pybtex-format mccoy/workflow/report/bibliography.bib mccoy/workflow/report/formatted-bibliography.html
```

Comment:
Will the user ever need to run this script? If not then can this be moved into the noxfile (that is where all dev tasks are currently kept).

What if they use `mccoy create --copy-workflow` and then want to add to the bibliography? How would the user do that?

Reply:
yes, definitely. that is much better

Reply:
@smutch - i'm not sure how to use the noxfile. Do you have time to look at it?

The bibliography thing is just hardcoded into the report template directory. I think it will be better when my snakemake PR is folded in to the main snakemake project and then we can switch to using that. That will be more flexible and new bibliography files can be added.

Reply:
Done. You can now format the bibliography using `nox -e format_bib`. I've also removed the pybtex dependency from pyproject.toml.